src/app/api/feedback/route.js

```diff
@@ -0,0 +1,59 @@
+import { NextResponse } from 'next/server';
+import { z } from 'zod';
+import { prisma } from '../../../lib/db';
+
+const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+const phoneRegex = /^\+380\d{9}$/;
+
+const Feedback = z
+  .object({
+    name: z
+      .string({
+        required_error: 'Name is required',
+        invalid_type_error: 'Name must be a string',
+      })
+      .max(128, {
+        message: 'Name must not be longer than 128 characters',
+      }),
+    phone: z
+      .string({
+        required_error: 'Phone number is required',
+        invalid_type_error: 'Phone number must be a string',
+      })
+      .refine(val => phoneRegex.test(val), {
```

Comment:
What do you think about trimming the value first and then .test it again the phoneRegex? Same with email.

![Screenshot from 2024-02-03 21-01-40](https://github.com/keenethics/mayak/assets/103147058/91b4e05b-f8e7-4419-b32a-16f140cc54db)